sql/moz-fx-data-shared-prod/search_terms_derived/suggest_impression_sanitized_v3/query.sql

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
This query needs to be a join of the impressions table and the sanitized output it's very similar to this here https://github.com/mozilla/bigquery-etl/blob/main/sql/moz-fx-data-shared-prod/search_terms_derived/suggest_impression_sanitized_v2_external/query.sql

With the exception of filtering against the know keyword list in the `remotesettings_suggestions_v1` table.

In psuedo code it's something like:

```sql

with impressions as (
  select 
  -- a bunch of stuff from the merino impressions table
  from `contextual_services_stable.quicksuggest_impression_v1`
  where date(submission_date) = @submission_date
), 
sanitized_queries as (
  select *
  from `search_terms_derived.merino_log_sanitized_v3`
  where date(timestamp) = @submission_date
)
select * 
from sanitized_queries
join impressions using (request_id)
```